nemo_skills/dataset/mmlu-pro/__init__.py

```diff
@@ -12,7 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-bs4
-evalplus @ git+https://github.com/evalplus/evalplus@c91370f
-pyyaml
-requests
+
+# settings that define how evaluation should be done by default (all can be changed from cmdline)
+
+PROMPT_CONFIG = 'generic/multichoice'
+DATASET_GROUP = 'multichoice'
+METRICS_TYPE = 'math'  # TODO: update this?
+DEFAULT_EVAL_ARGS = (
+    "++eval_type=mmlu "
+    "++eval_config.parse_func=llama "
+)
+
+
+DEFAULT_GENERATION_ARGS = ""
```

Comment:
```suggestion
DEFAULT_GENERATION_ARGS = "++split=test_llama"
```
You could also add `++split=test_llama` to here so that it's used by default and matches the parse func